web/src/components.ts

```diff
@@ -41,10 +41,13 @@ export function toggle(opts: {
     for (const [i, value] of opts.values.entries()) {
         // create a tab with a tab-id so they don't have to be referenced
         // by text value which can be inconsistent.
-        const $tab = $("<div>")
+        const $tab_parent = $("<div>")
```

Comment:
The tooltip is not shown when the tab is disabled because `pointer-events` is set to `none`. We need a wrapper when a button or input is disabled because in that case the `disabled` attribute is set to `true` while here we just add `disabled ` class.

I guess the `pointer-events` property was set to `none` to make sure that the cursor on hovering over the disabled tab is not pointer as the code for not doing anything on clicking on disabled tab is in JS.

I guess we can instead set the `cursor` property for disabled tab and remove the `pointer-events` property and then there would be no need for this commit.

Reply:
Sure, that simplifies things a lot.